### PR TITLE
Minimal k8s version requirement

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubali@redhat.com>

### CODE REVIEW

File: content/docs/2.0/deploy.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ We provide a few approaches to deploy KEDA runtime in your Kubernetes clusters:
 - [Operator Hub](#operatorhub)
 - [YAML declarations](#yaml)
 
+> 💡 **NOTE:** KEDA requires Kubernetes cluster version 1.16 and higher
```
Comment (tomkerkhove) [inline content/docs/2.0/deploy.md:11]: Should we add this to FAQ as well?

**Review by tomkerkhove — APPROVED** [review]

File: content/docs/2.0/deploy.md (hunk 1 above)
Comment (zroubalik) [reply content/docs/2.0/deploy.md:11]: Great idea, I am gonna add that

Comment (zroubalik) [reply content/docs/2.0/deploy.md:11]: https://github.com/kedacore/keda-docs/pull/273

Comment (zroubalik) [reply content/docs/2.0/deploy.md:11]: this needs to be solved first^ (versioned FAQ)

Comment (zroubalik) [reply content/docs/2.0/deploy.md:11]: @tomkerkhove added info to the FAQ, PTAL

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**